### PR TITLE
Script for summarizing PRs

### PR DESCRIPTION
**Pull request by zmoon**
Regarding #258.

Currently a script with a few command-line options; it prints markdown to stdout.

* Is this a good place to put it? (`_scripts/`)
* It would be easy to add more data about the PRs, such as author or date

### CODE REVIEW

**Comment by certik** [comment]
I think that's a good start. Thanks!

**Comment by zmoon** [comment]
@certik anything else you want to see it do at this time?

File: _scripts/prs.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    # GitHub connection
+    if token is None:
+        token = os.environ.get("GITHUB_TOKEN")
+    if token is None:
+        print("token not set. GitHub API may complain")
+    g = github.Github(token, per_page=100)
+    org = g.get_organization("fortran-lang")
+
+    for reponame in ["stdlib", "fpm"]:
```
Comment (awvwgk) [inline _scripts/prs.py:50]: The webpage is usually in the newsletter as well.
```suggestion
    for reponame in ["fortran-lang.org", "stdlib", "fpm"]:
```
There are plenty of other repositories, like `setup-fpm`, `stdlib-cmake-example`, `fpm-haskell` in the @fortran-lang namespace with low activity I would usually associate with the parent project if something is going on there.

Comment (zmoon) [reply _scripts/prs.py:50]: Hmm so would it be more helpful for their PRs to be in the list for the parent project or have their own separate lists?

Comment (awvwgk) [reply _scripts/prs.py:50]: Given that those projects are usually on a rather low activity level, I don't think they warrant their own section. In the past we announced those projects in the section of their respective parent project, I'm not sure if we picked up all setup-fpm updates in the past in the fpm section, actually.

Comment (zmoon) [reply _scripts/prs.py]: Ok, I added this capability in https://github.com/fortran-lang/fortran-lang.org/pull/261/commits/7de8b764e4e16a711e2bb1516720aa1f2e751d70, let me know what you think. You can see it working with `--month 2021-03` for example

**Comment by certik** [comment]
Can you show an example how the output looks like from this script?

**Comment by zmoon** [comment]
<details>
<summary>Yeah, here for 2021-03:</summary>

```
# fortran-lang.org

## merged

* [#236](https://github.com/fortran-lang/fortran-lang.org/pull/236): add dl_poly_4 to packages
* [#229](https://github.com/fortran-lang/fortran-lang.org/pull/229): Correct value of pi in quickstart mini-book
* [#226](https://github.com/fortran-lang/fortran-lang.org/pull/226): Added DelaunaySparse to package list
* [#225](https://github.com/fortran-lang/fortran-lang.org/pull/225): Update Authors in GSoC post
* [#223](https://github.com/fortran-lang/fortran-lang.org/pull/223): GSoC announcement draft
* [#222](https://github.com/fortran-lang/fortran-lang.org/pull/222): Avoid unclear formulation in contributing guide
* [#221](https://github.com/fortran-lang/fortran-lang.org/pull/221): Add information about free compiler versions
* [#207](https://github.com/fortran-lang/fortran-lang.org/pull/207): Correct subtitle of setting up your os
* [#216](https://github.com/fortran-lang/fortran-lang.org/pull/216): Improve tags
* [#215](https://github.com/fortran-lang/fortran-lang.org/pull/215): Newsletter for March 2021

## WIP

* [#201](https://github.com/fortran-lang/fortran-lang.org/pull/201) (WIP): Draft: Internationalization for fortran-lang

# stdlib

## merged

* [#346](https://github.com/fortran-lang/stdlib/pull/346): Added to_lower, to_upper, reverse and to_title function to stdlib_string_type.f90 file
* [#362](https://github.com/fortran-lang/stdlib/pull/362): Fix wording in style guide for dimension attribute
* [#356](https://github.com/fortran-lang/stdlib/pull/356): Add test_string_functions to CMakeLists and fix to_title test
* [#352](https://github.com/fortran-lang/stdlib/pull/352): Added TOC to README
* [#350](https://github.com/fortran-lang/stdlib/pull/350): Style: Remove tabs from code base
* [#320](https://github.com/fortran-lang/stdlib/pull/320): Implement non-fancy functional string type

## WIP

* [#363](https://github.com/fortran-lang/stdlib/pull/363) (WIP): Added sort to stdlib_string_type.f90 file
* [#353](https://github.com/fortran-lang/stdlib/pull/353) (WIP): Initial checkin for a module for tolerant comparison of reals
* [#333](https://github.com/fortran-lang/stdlib/pull/333) (WIP): Provide abstract base class for a string object
* [#313](https://github.com/fortran-lang/stdlib/pull/313) (WIP): legendre polynomials and gaussian quadrature
* [#311](https://github.com/fortran-lang/stdlib/pull/311) (WIP): String list new
* [#286](https://github.com/fortran-lang/stdlib/pull/286) (WIP): Probability Distribution and Statistical Functions -- Beta Distribution Module
* [#278](https://github.com/fortran-lang/stdlib/pull/278) (WIP): Probability Distribution and Statistical Functions -- Gamma Distribution Module
* [#276](https://github.com/fortran-lang/stdlib/pull/276) (WIP): Probability Distribution and Statistical Functions -- Exponential Distribution Module
* [#273](https://github.com/fortran-lang/stdlib/pull/273) (WIP): Probability Distribution and Statistical Functions -- Normal Distribution Module
* [#272](https://github.com/fortran-lang/stdlib/pull/272) (WIP): Probability Distribution and Statistical Functions -- Uniform Distribution Module
* [#189](https://github.com/fortran-lang/stdlib/pull/189) (WIP): Initial implementation of COO / CSR sparse format
* [#157](https://github.com/fortran-lang/stdlib/pull/157) (WIP): Update CMAKE files

# fpm

## merged

* [#420](https://github.com/fortran-lang/fpm/pull/420): Phase out Haskell fpm
* [#412](https://github.com/fortran-lang/fpm/pull/412): Fix for Issue #396 - Duplicate module definitions
* [#419](https://github.com/fortran-lang/fpm/pull/419): Split workflow for Haskell and Fortran fpm
* [#417](https://github.com/fortran-lang/fpm/pull/417): Bump version for new release
* [#413](https://github.com/fortran-lang/fpm/pull/413): Add: omp_lib to intrinsic modules list
* [#409](https://github.com/fortran-lang/fpm/pull/409): Give Programs Access to Code in Subdirectories
* [#404](https://github.com/fortran-lang/fpm/pull/404): correct join for null input
* [#414](https://github.com/fortran-lang/fpm/pull/414): Add few important links
* [#377](https://github.com/fortran-lang/fpm/pull/377): Add explicit include-dir key to manifest
* [#407](https://github.com/fortran-lang/fpm/pull/407): Fix typo in fpm-build synopsis
* [#390](https://github.com/fortran-lang/fpm/pull/390): Implement --flag option for Fortran fpm
* [#402](https://github.com/fortran-lang/fpm/pull/402): Add fpm description and reorganize the README intro
* [#397](https://github.com/fortran-lang/fpm/pull/397): Add Conda install instructions to the README
* [#386](https://github.com/fortran-lang/fpm/pull/386): Replace deprecated flags in debug_fortran option
* [#398](https://github.com/fortran-lang/fpm/pull/398): Minor fix: for setting executable link libraries.
* [#5](https://github.com/fortran-lang/setup-fpm/pull/5) (`setup-fpm`): Correctly document option to use Haskell fpm version
* [#1](https://github.com/fortran-lang/fpm-haskell/pull/1) (`fpm-haskell`): Prune repository for Haskell only

## WIP

* [#364](https://github.com/fortran-lang/fpm/pull/364) (WIP): plugin alpha version
```
</details>

**Comment by certik** [comment]
Thanks! I think this looks very good. Thank you for creating a script for this. I need to create a similar script for GitLab for LFortran. Last month we had 98 merge requests and I it was too much work to do it by hand. :)

**Review by certik — APPROVED** [review]

**Review by LKedward — APPROVED** [review]

**Review by milancurcic — APPROVED** [review]
Thank you!

**Pull request merged**